spec.html

```diff
@@ -33531,7 +33531,7 @@
           1. Assert: Type(_value_) is Number.
           1. Let _block_ be _arrayBuffer_.[[ArrayBufferData]].
           1. Assert: _block_ is not *undefined*.
-          1. If _isLittleEndian_ is not present, set _isLittleEndian_ to the value of the [[LittleEndian]] internal slot of the surrounding agent's Agent Record.
+          1. If _isLittleEndian_ is not present, set _isLittleEndian_ to the value of the [[LittleEndian]] field of the surrounding agent's Agent Record.
```

Comment:
these are called "internal slots" throughout the spec (`ag 'internal slot' | wc -l` gives me 365 occurrences, `ag ']] field' | wc -l` gives me only 41) - why change it to "field" here?